BrainPortal/cbrain_plugins/cbrain-plugins-base/userfiles/csv_file/csv_file.rb

```diff
@@ -27,7 +27,7 @@ class CSVFile < TextFile
 
   Revision_info=CbrainFileRevision[__FILE__] #:nodoc:
 
-  has_viewer :name => 'CSV Viewer', :partial  => :csv_file, :if => :csv_is_viewable?
+  has_viewer :name => 'CSV Viewer', :partial  => :csv_file, :if => Proc.new { |u| u.size_allows_viewing_400_000 }
```

Comment:
The entire point of the exercise was to simplify the job of the developer when configuring the `if` callback... and you still use a Proc... ? Did you forget?